cpu/cpu_plan9_test.go

```diff
@@ -35,7 +35,7 @@ var timesTests = []struct {
 func TestTimesPlan9(t *testing.T) {
 	for _, tt := range timesTests {
 		t.Run(tt.mockedRootFS, func(t *testing.T) {
-			t.Setenv("HOST_ROOT", filepath.Join("testdata/plan9", tt.mockedRootFS))
+			t.Setenv("HOST_ROOT", filepath.Join("testdata", "plan9", tt.mockedRootFS))
```

Comment:
Is there a linter in go-critic that reported it?

I was looking for it a few months ago